### ISSUE TITLE
[BUG] Unknown scheme for WebSocket.listenerCount

### ISSUE THREAD

**Issue by LanderBeeuwsaert** [microsoft/playwright#4445]
**Context:**
- Playwright Version: [what Playwright version do you use?]
1.6.1

- Operating System: [e.g. Windows, Linux or Mac]
windows

- Node.js version: [e.g. 10.12, 10.14]
12.18.2

- Browser: [e.g. All, Chromium, Firefox, WebKit]
Chromium

- Extra: [any specific details about your environment]

**Code Snippet**

Help us help you! Put down a short code snippet that illustrates your bug and
that we can run and debug locally. For example:

```javascript
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch({ headless: false, slowMo: 50, devtools: true });
  const context = await browser.newContext({ viewport: { width: 1500, height: 1500 } });
  const page = await context.newPage();
  await page.goto('http://localhost:4200/');
  const handle = await page.$('text="GET STARTED"');
  await handle.click();

  await browser.close();
})();
```

**Describe the bug**
If I run this piece of code, I get the following error:

```
(node:23332) UnhandledPromiseRejectionWarning: Error: Unknown scheme for WebSocket.listenerCount
    at validateParams (C:\Lander\WS\acro-companion\e2e-testing\playwright\node_modules\playwright\lib\client\connection.js:248:15)
    at Connection.sendMessageToServer (C:\Lander\WS\acro-companion\e2e-testing\playwright\node_modules\playwright\lib\client\connectio
n.js:72:67)
    at Proxy.<anonymous> (C:\Lander\WS\acro-companion\e2e-testing\playwright\node_modules\playwright\lib\client\channelOwner.js:54:53)
    at Proxy.EventEmitter.emit (domain.js:476:32)
    at Connection.dispatch (C:\Lander\WS\acro-companion\e2e-testing\playwright\node_modules\playwright\lib\client\connection.js:118:25
)
    at Immediate.<anonymous> (C:\Lander\WS\acro-companion\e2e-testing\playwright\node_modules\playwright\lib\inprocess.js:42:85)
    at processImmediate (internal/timers.js:456:21)
    at process.topLevelDomainCallback (domain.js:137:15)
(node:23332) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an asyn
c function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unha
ndled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejection
s_mode). (rejection id: 1)
(node:23332) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not
 handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by aslushnikov** [comment]
@LanderBeeuwsaert i *think* this is addressed by https://github.com/microsoft/playwright/pull/4435. Can you please try `playwright@next` and see if it works? 

The fix with https://github.com/microsoft/playwright/pull/4435 will be released shortly.

**Comment by LanderBeeuwsaert** [comment]
@aslushnikov 
Hey, I just tested with "playwright": "1.7.0-next.1605637580878", and I have the same issue.

**Comment by aslushnikov** [comment]
@LanderBeeuwsaert this should be fixed now with #4495! Could you please verify by installing `playwright@next`?

**Comment by LanderBeeuwsaert** [comment]
Hi @aslushnikov , done and confirmed! Thanks, closing.
